src/test/java/com/gfl/client/model/ProxyCredentialsTest.java

```diff
@@ -0,0 +1,93 @@
+package com.gfl.client.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProxyCredentialsTest {
```

Comment:
the same situation with `@Size` and `@NotBlank`

    @NotBlank(message = "must not be blank")
    @Size(min = 6, max = 100, message = "must be between {min} and {max} characters")